package.json

```diff
@@ -356,7 +356,7 @@
     "js-yaml": "^3.14.1",
     "jpeg-js": "0.4.4",
     "qs": "6.9.7",
-    "rollup": "^3.11.0"
+    "rollup": "^3.14.0"
```

Comment:
Thought this would fix the class name mangling… https://github.com/rollup/rollup/pull/4827